src/Identity/Extensions.Core/src/UserManager.cs

```diff
@@ -714,7 +714,7 @@ public virtual async Task<bool> CheckPasswordAsync(TUser user, string password)
             var success = result != PasswordVerificationResult.Failed;
             if (!success)
             {
-                Logger.LogWarning(0, "Invalid password for user {userId}.", await GetUserIdAsync(user));
+                Logger.LogWarning(LoggerEventIds.InvalidPassword, "Invalid password for user {userId}.", await GetUserIdAsync(user));
```

Comment:
I don't think we can just change log ids like this can we?  @ajcvickers @blowdart is there some kind of guarantee with event ids being stable generally in logs?

Reply:
Yes, they are expected to remain stable.

Reply:
@Kahbazi so I guess if you are going to move to a variable for these, you are going to have to ensure that the order/values match.